tests/db/TransactionTest.php

```diff
@@ -152,7 +152,7 @@ public function testTransactions()
                     ->set(['id' => 4, 'FOO' => 'bar', 'name' => 'Jane', 'surname' => 'Doe', 'retired' => 0])
                     ->insert();
             });
-        } catch (\PDOException $e) {
+        } catch (\Exception $e) {
```

Comment:
yeah it's test script and need to have Exception instead of PDOException because now Expresion->execute catches PDOException and throws its  own exception insted with some additional info.